packages/dcos-integration-test/extra/test_rlimit.py

```diff
@@ -0,0 +1,33 @@
+import subprocess
+import uuid
+
+
+def test_if_rlimits_can_be_used(cluster):
+    """This test verifies that rlimits can be used.
+
+    Since marathon does not support rlimits yet we use `mesos-execute` as
+    scheduler. We run a job for which we specify an unlimited `RLIMIT_CORE`.
+    The job then examines the actual limit, and returns a failure if another
+    value is found."""
+
+    name = 'test-rlimits-{}'.format(uuid.uuid4().hex)
+
+    argv = [
+        '/opt/mesosphere/bin/mesos-execute',
+        '--rlimits={"rlimits": [{"type":"RLMT_CORE"}]}',
+        '--master=leader.mesos:5050',
+        '--name={}'.format(name),
+        '--command=ulimit -c | grep -q unlimited',
+        '--shell=true',
+        '--env={"LC_ALL":"C"}']
+
+    output = subprocess.check_output(
+        argv,
+        stderr=subprocess.STDOUT,
+        universal_newlines=True)
+
+    expected_output = \
+        "Received status update TASK_FINISHED for task '{name}'".format(
+            name=name)
+
+    assert expected_output in output
```

Comment:
It is now clear to me how this tests what is described in the docstring:

> The job then examines the actual limit, and returns a failure if another value is found

This just makes sure that TASK_FINISHED was returned somewhere in the output. When a task finishes with an nonzero exit code, does it produce a different string? From a layperson's view, it would follow that a non-zero exit would still get a "TASK_FINISHED" message